core/state/trie_prefetcher.go

```diff
@@ -18,13 +18,18 @@ package state
 
 import (
 	"sync"
+	"sync/atomic"
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/log"
 	"github.com/ethereum/go-ethereum/metrics"
 )
 
-const abortChanSize = 64
+const (
+	abortChanSize                 = 64
+	parallelTriePrefetchThreshold = 100
+	parallelTriePrefetchCapacity  = 200
```

Comment:
Will `parallelTriePrefetchCapacity` be too big? How to measure whether it is a reasonable value?

Reply:
I tried 50, 100 before, but smaller capacity could cause more child goroutine to be created.
Since there are already lots of trie prefetch routine right now, I would prefer to limit the child goroutine number.
The PR is mainly for handling big blocks, 200 could be enough. But it is ok, I will double confirm,